components/index.js

```diff
@@ -3,6 +3,8 @@ export { default as Dashicon } from './dashicon';
 export { default as FormToggle } from './form-toggle';
 export { default as IconButton } from './icon-button';
 export { default as Panel } from './panel';
+export { default as PanelHeader } from './panel/header';
+export { default as PanelBody } from './panel/body';
```

Comment:
Do we want to treat nested components the same as top-level? Wondering if either `Panel.Body` or `import PanelBody from 'components/panel/header';` are better alternatives.